opal/mca/accelerator/Makefile.am

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2014      Intel, Inc.  All rights reserved.
+# Copyright (c) 2022      Amazon.com, Inc. or its affiliates.
+#                         All Rights reserved.
+# $COPYRIGHT$
+#
+# Additional copyrights may follow
+#
+# $HEADER$
+#
+
+AM_CPPFLAGS = $(LTDLINCL) $(accelerator_CPPFLAGS)
```

Comment:
You don't need this line at all.  You don't need the `$(LTDLINCL)` because we don't use libltdl for its faked dlopen support.  you don't need `$(accelerator_CPPFLAGS)` because you don't have a framework-level configure.m4 snippet to ever set `$(accelerator_CPPFLAGS)`.

Reply:
Line Removed.